integration_tests/test_snap.py

```diff
@@ -131,8 +131,8 @@ def test_error_with_unexistent_build_package(self):
         exception = self.assertRaises(
             subprocess.CalledProcessError, self.run_snapcraft, 'snap')
         expected = (
-            'Could not find all the "build-packages" required in '
-            'snapcraft.yaml\n')
+            "Could not find a required package in 'build-packages': "
+            '"The cache has\nno package named \'inexistent-package\'"\n')
```

Comment:
The newline in here is throwing me. Why is it here and not in the other test below?